py2puml/cli.py

```diff
@@ -16,7 +16,7 @@ def run():
 
     argparser = ArgumentParser(description='Generate PlantUML class diagrams to document your Python application.')
 
-    argparser.add_argument('-v', '--version', action='version', version='py2puml 0.7.2')
+    argparser.add_argument('-v', '--version', action='version', version='py2puml 0.8.0')
```

Comment:
Does it make sense to source this directly from `pyproject.toml`?

Reply:
An alternative for version numbering would be to use this poetry plugin: https://pypi.org/project/poetry-dynamic-versioning/

Reply:
> Does it make sense to source the version number directly from `pyproject.toml` or fetch it via the `pkg_resources` package?

I wish I could use the `pyproject.toml` file as a single source of truth for the production code (like in the metadata of this CLI arg parser). Unfortunately, only the contents of `py2puml/py2puml/` gets included in the package and installed in the virtual environment when people `pip install py2puml`. I thus need to check the consistency of this piece of metadata the other way around:

- change the version in pyproject.toml
- change this textual metadata
- so that the unit test (which parses the `pyproject.toml` file) check the consistency

It is neither optimal nor satisfying but it is no big deal.

Reply:
> An alternative for version numbering would be to use this poetry plugin: https://pypi.org/project/poetry-dynamic-versioning/

Yes, it is on the roadmap. That is why I started using github actions for the continuous integration of this project. But it would be with a specific pull request